src/app/views/sidebar/resource-explorer/CommandOptions.tsx

```diff
@@ -26,13 +28,32 @@ const CommandOptions = (props: ICommandOptions) => {
     setIsOpen(open);
   }
 
+  const itemStyles: Partial<IContextualMenuItemStyles> = {
```

Comment:
Could we have the styles in their own file?

Reply:
Sure